renormalizer/cv/spectra_cv.py

```diff
@@ -103,6 +104,8 @@ def __init__(
         else:
             self.cv_mps = cv_mps
         
+        self.cv_mps.compress_config = CompressConfig(CompressCriteria.fixed, max_bonddim=m_max)
```

Comment:
Is `self.m_max` still used? (resolve directly if yes)

Reply:
Yes. It is used in initialized random state and select basis in finite temperature algorithm.
It is not hard to replace all of them with information from compres_config.
I didn't do it because of the time. I just make sure that all the tests are passed. :cry:

Reply:
OK then no problem